py/vtdb/db_object.py

```diff
@@ -1,26 +1,24 @@
-"""Module containing the base class for database classes and decorator for db method.
-
-The base class DBObjectBase is the base class for all other database base classes.
-It has methods for common database operations like select, insert, update and delete.
-This module also contains the definition for ShardRouting which is used for determining
-the routing of a query during cursor creation.
-The module also has the db_class_method decorator and db_wrapper method which are
-used for cursor creation and calling the database method.
+"""Base class for database classes and decorator for db method.
+
+The base class DBObjectBase is the base class for all other database
+base classes.  It has methods for common database operations like
```

Comment:
Text has double spaces after the period.

Please change them to a single space.